app/views/system/admin/announcements/edit.html.slim

```diff
@@ -1,3 +1,4 @@
-- add_breadcrumb format_inline_text(@announcement.title)
-= page_header
-= render 'form'
+div#system-admin-component
+/ - add_breadcrumb format_inline_text(@announcement.title)
```

Comment:
Actually in the announcements controller, we don't even need the `new` and `edit` actions anymore, since only `create` and `update` is called. Should I remove them too? (right now going to /admin/announcements/new is blank, but it might make more sense to 404)

Reply:
Yes should do that thanks